asynq/tests/test_active_task.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 Quora, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from asynq import async, scheduler
+from qcore.asserts import assert_is, assert_is_not
+
+
+@async()
+def outer_async():
+    """Test that we get the correct active task from the scheduler.
+
+    Previously, there was a bug such that when we called a task synchronously from an async
```

Comment:
this context is not necessary. You can just say that this test is to make sure that the scheduler correctly maintains the active task even when the execution of one task gets interrupted by a synchronous call to another async function.